pkg/download/handler.go

```diff
@@ -41,7 +41,7 @@ func RegisterHandlers(app *buffalo.App, opts *HandlerOpts) {
 	}
 	noCacheMw := middleware.CacheControl("no-cache, no-store, must-revalidate")
 
-	listHandler := LogEntryHandler(ListHandler, opts)
+	listHandler := LogEntryHandler(ListHandlerBasic, opts)
```

Comment:
That's an interesting idea. I actually thought we might be able to go ahead and use `RegisterHandlersV2` because buffalo exposes the underlying `*mux.Router` via the `Muxer` method. It would be awesome if we could slowly start defining the mux routes, passing `app.Muxer()` into this V2 method, but I can't seem to get buffalo to pick up the mux route. It just returns a 404. 

In the meantime we can just not call the V2 method, but it would be nice to start using the new handlers as they're defined as much as possible.

Reply:
Scratch that, I was actually able to get it working. I updated the PR to show running both of these methods side-by-side so that we can actually use the new handlers we're defining, as we define them.